pkg/cmd/cli/install/install.go

```diff
@@ -90,6 +91,7 @@ func (o *InstallOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&o.UseRestic, "use-restic", o.UseRestic, "create restic deployment. Optional.")
 	flags.BoolVar(&o.Wait, "wait", o.Wait, "wait for Velero deployment to be ready. Optional.")
 	flags.DurationVar(&o.DefaultResticMaintenanceFrequency, "default-restic-prune-frequency", o.DefaultResticMaintenanceFrequency, "how often 'restic prune' is run for restic repositories by default. Optional.")
+	flags.Var(&o.Plugins, "plugins", "Plugin container images to install into the Velero Deployment. Optional.")
```

Comment:
Isn't this supposed to be required?

Reply:
Or should I change it to required on my PR that extracts the cloudproviders?

Reply:
I can change it now if you'd like; I actually need to add a change log anyway.

It's a bit awkward either way at the moment - if it's required now, we basically have to add a dummy plugin image to get `velero install` to work on master, as the cloud provider plugins aren't present yet. 

If it's optional once the plugins are out of tree, then `velero install` could install a deployment without any object storage plugins present, which would mean it's nonfunctional.

Reply:
Let me change it to required as part of the PR that extracts the providers then. That will only go in once we have the plugin repos and their images.